### ISSUE TITLE
import SelectDateWidget error

### ISSUE THREAD

**Issue by tartarini** [dyve/django-bootstrap3#375]
**problem**
working with both git versions of django (2.0.dev20170222001300) and django-bootstrap3 (branch dev commit 465e61570453d7633a4bb122b22ee5334ad378d8) I encountered this error:

```
    django-bootstrap3/bootstrap3/renderers.py", line 14, in <module>
    from django.forms.extras import SelectDateWidget
    ImportError: No module named 'django.forms.extras'

```

**Solution**

SelectDateWidget is now in django.forms.widgets so editing renderers.py would work for me:

```
#from django.forms.extras import SelectDateWidget
from django.forms.widgets import SelectDateWidget
```

**Comment by nim65s** [comment]
cf. #374 :)

**Comment by dyve** [comment]
This is fixed with the changes in #374.
